### ISSUE TITLE
pip install fail

### ISSUE THREAD

**Issue by melMass** [mdboom/pytoshop#1]
* psdwriter version:
* Python version: 3.6
* Operating System: OSX 10.11

### Description

Installing from pip fail with the following message:
`ValueError: 'pytoshop/packbits.pyx' doesn't match any files`

### What I Did

Using the following works well.
```
pip install git+https://github.com/mdboom/pytoshop.git
```

**Comment by mdboom** [comment]
Thanks for the report.  PR #2 should hopefully address this, and once tested, I'll make a new release.

**Comment by cprogrammer1994** [comment]
I get `ImportError: No module named 'Cython'`

```
pip install git+https://github.com/mdboom/pytoshop.git
```

I'll install it manually then retry.

**Comment by cprogrammer1994** [comment]
Built on windows with:

- Python35-32bit
- Python35-64bit
- Python36-32bit
- Python36-64bit

[log](https://gist.github.com/cprogrammer1994/d2e645ab180af36884c9ad49f4ac713c)

**Comment by mdboom** [comment]
@cprogrammer1994: Thanks for the report.  Your bug is subtley different (involving not being able to read the package metadata without Cython installed first).  But it's simplest just to add the fix to #2 as well.

**Comment by cprogrammer1994** [comment]
I get the error because cython is imported before the `setup()` is called.

- [import cython on line 6](https://github.com/mdboom/pytoshop/blob/master/setup.py#L6)
- [the setup is called on line 31](https://github.com/mdboom/pytoshop/blob/master/setup.py#L31)

I don't know what would be a solution for this. Maybe a `generator` for `ext_modules=...` can solve that.

**Comment by mdboom** [comment]
I have a possible solution in 0c318518.  This works because pip first calls `setup.py egg_info` to get the metadata (before Cython is there), then installs Cython, then calls `setup.py install` to install.

**Comment by mdboom** [comment]
Fixed by #2, and new 0.4.1 release posted to PyPI
